### PR TITLE
German Locale

### PR DESCRIPTION
**Pull request by MetalMatze**
This is a first draft for german locales. I'm not 100% with all of these translation.

I can't come up on how to translate `"audio-device-communications": "System Default Communications",`.

To german speaking people:
Wenn du Deutsch spricht, dann lies dir diese Vorschläge zur Übersetzung bitte durch. Hier sind vermutlich Fehler zufinden oder eine unglückliche Wortwahl.

### CODE REVIEW

**Comment by MarshallOfSound** [comment]
@MetalMatze Does it become easier to translate if I rephrase it to say

"Default System Communication Device"

File: src/_locales/de-DE.json [hunk 1]
```diff
@@ -0,0 +1,94 @@
+{
+  "audio-device-communications": "System Default Communications",
+  "audio-device-default": "Standardeinstellung",
+  "audio-device-unknown": "Unbekanntes Gerät",
```
Comment (MCManuelLP) [inline src/_locales/de-DE.json]: Maybe make that:
    "audio-device-communications": "Standard Kommunikations Gerät"
    "audio-device-default": "Standard Gerät"
For consistency....

**Comment by MetalMatze** [comment]
I think I'm missing the context, so that I can translate it properly. But honestly I haven't looked at the code. Maybe I'll do it later 😉

**Comment by MarshallOfSound** [comment]
@MetalMatze Please rename the file to `de.json` as per #1003

File: src/_locales/de.json [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+  "lastfm-login-error": "Du hast den Autorisierungsvorgang verlassen. Wenn das ein ein Fehler war, versuche es nochmal",
+  "lastfm-login-not-authorized": "Nicht autorisiert",
+  "lastfm-logout-button-text": "Von Last.FM abmelden",
+
+  "lyrics-failed-message": "Entschuldige, aber wir konnten keinen Liedtext finden",
+  "lyrics-loading-message": "Lade Liedtext...",
+  "lyrics-no-song-message": "Aktuell spielt kein Titel",
+
+  "message-pausing-after-song": "Pausieren nach diesem Titel.",
+  "message-pausing-after-song-button": "Nicht nach nach diesem Titel pausieren",
```
Comment (petschekr) [inline src/_locales/de.json]: Shouldn't this read `Nicht nach diesem Titel pausieren`?

Comment (MetalMatze) [inline src/_locales/de.json]: Yes, indeed. I just wrote that whole thing from top to bottom and read it once afterwards. Thanks!

**Comment by berryelectronics** [comment]
How about "Standard Audiogeräte" and "Standard Kommunikationsgerät". I think thats the translation Windows uses for these devices

File: src/_locales/de-DE.json (hunk 1 above)
Comment (berryelectronics) [inline src/_locales/de-DE.json]: @MCManuelLP rather "Standardgerät" and "Standardkommunikationsgerät" . Would line up perfectly with MS's naming

File: src/_locales/de-DE.json [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+  "playback-label-next-track": "Nächster Titel",
+  "playback-label-play-pause": "Abspielen / Pausieren",
+  "playback-label-previous-track": "Vorheriger Titel",
+  "playback-label-stop": "Stop",
+  "playback-label-thumbs-down": "Daumen hoch",
+  "playback-label-thumbs-up": "Daumen runter",
+  "playback-label-volume-down": "Leiser",
+  "playback-label-volume-up": "Lauter",
+
+  "settings-option-custom-theme": "Eigenes Thema",
```
Comment (berryelectronics) [inline src/_locales/de-DE.json]: "Eigenes Farbschema" hört sich meiner Meinung nach besser an.

**Comment by MetalMatze** [comment]
Please check again. I'm not 100% sure if that's what you in intended.

**Comment by bernikr** [comment]
A bit about Windows audio devices:
In the sound settings you can set an device as system default ("Standardgerät") and optionally another device as communication default ("Standardkommunikationsgerät"). Windows will use the communication default for things like Skype, while the standard device is used for anything else.

To be consistent with the translation of Windows I would change `audio-device-communications` to `Standardkommunikationsgerät` and `audio-device-default` to `Standardgerät`.

Besides that your translation looks all right.

**Comment by MetalMatze** [comment]
I don't have Windows. What about that? 😊 But nevertheless I'll update that.

**Comment by bernikr** [comment]
I honestly don't know if this difference in devices exists on any other platform. I just assumed it is a very windows-y thing and looked for the official translations.

**Comment by MarshallOfSound** [comment]
@bernikr From a technical point of view the `communications` and `default` devices only appear on Windows machines (I think).

On OS X and Linux names are pulled correctly from the system in the correct language.

File: src/_locales/de-DE.json (hunk 1 above)
Comment (ChrisB9) [inline src/_locales/de-DE.json]: I think, in ›Standardkommunikationsgerät‹ should be a ›system‹ anywhere to show the user where the settings are coming from..

File: src/_locales/de-DE.json [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+
+  "message-pausing-after-song": "Pausieren nach diesem Titel.",
+  "message-pausing-after-song-button": "Nicht nach nach diesem Titel pausieren",
+  "message-uncaught-error": "Es gab einen Fehler innerhalb von GPMDP.",
+  "message-uncaught-error-button": "Klicke um den Fehler bei GitHub zu melden",
+
+  "modal-confirmTray-title": "Ein kurzer Hinweis",
+  "modal-confirmTray-content": "Das Hauptfenster zu schließen (was du gerade versucht hast) wird den Player in den Infobereich minimieren. Um dieses Verhalten zu ändern gehe in die \"Desktop-Einstellungen\".",
+  "modal-confirmUpdate-title": "Aktualisierung verfügbar",
+  "modal-confirmUpdate-content": "Woo hoo, es sieht so aus, als gäbe es eine Aktualisierung für Google Play Music Desktop Player.  Diese Aktualisierung bringt neue Extras, Fehlerbehebungen und Weiteres.  Für das beste Erlebnis empfehlen wir so schnell wie möglich zu aktualisieren. <br/>Wir hoffen dir gefällt diese Aktualisierung!",
```
Comment (ChrisB9) [inline src/_locales/de-DE.json]: `<br/>Wir hoffen dir gefällt diese Aktualisierung!"`
Muss heißten: Wir hoffen, dir gefällt [...]

Mehr dazu hier :) :
http://www.korrekturen.de/sprachleben/zeichensetzung/komma_nach_hoffe_und_denke.shtml

File: src/_locales/de-DE.json [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+  "modal-confirmUninstall-title": "Hier ist das Problem...",
+  "modal-confirmUninstall-content": "Wir haben gerade herausgefunden, dass noch eine alte Version von GPMDP installiert ist.  Wenn du gerade aktualisiert hast, ist es normal.  Wir versuchen nun die alte Version zu löschen.  Dies \"könnte\" Administrationsrechte benötigen. Bitte gib uns diese, damit wir aufräumen können.",
+  "modal-goToURL-title": "Gib die Google Play Music URL um dort hin zu gehen:",
+  "modal-welcome-title": "Willkommen zur Version",
+  "modal-welcome-content": "Wir hoffen dir gefällt diese Aktualisierung!",
```
Comment (ChrisB9) [inline src/_locales/de-DE.json]: Hier auch, siehe Kommentar weiter oben

File: src/_locales/de-DE.json [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+  "playback-label-stop": "Stop",
+  "playback-label-thumbs-down": "Daumen hoch",
+  "playback-label-thumbs-up": "Daumen runter",
+  "playback-label-volume-down": "Leiser",
+  "playback-label-volume-up": "Lauter",
+
+  "settings-option-custom-theme": "Eigenes Thema",
+  "settings-option-custom-theme-choose-color": "Wähle deinen Farbakzent",
+  "settings-option-custom-theme-choose-custom": "Oder wähle eine",
+  "settings-option-custom-theme-custom-color": "Eigene Farbe",
```
Comment (ChrisB9) [inline src/_locales/de-DE.json]: Optional:

das E von Eigene vielleicht klein, da es ja nach "Oder wähle eine " kommt.
Man könnte es auch groß lassen, damit es betont wird - bin mir hier nicht sicher ;)

File: src/_locales/de-DE.json (hunk 3 above)
Comment (ChrisB9) [inline src/_locales/de-DE.json]: Unter Linux wird ›Theme‹ auch mit Thema übersetzt. Aus Konsistenzgründen würde ich das also hier auch so lassen, wie es ist.

File: src/_locales/de-DE.json [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+  "settings-option-custom-theme-dark": "Dunkel",
+  "settings-option-custom-theme-light": "Hell",
+
+  "settings-option-enable-api": "Aktiviere die Playback API",
+  "settings-option-enable-api-details": "Dies erlaubt es anderen Applikationen mit GPMDP zu interagieren",
+  "settings-option-enable-system-borders": "Benutze Fensterumrandung des Systems",
```
Comment (ChrisB9) [inline src/_locales/de-DE.json]: Im Unity-Tweak-Tool heißt es ›Fensterrahmenthema‹, klingt das besser als Fensterumrandung?
(Ich kann mich mit Umrandung noch nicht so ganz anfreunden..)

File: src/_locales/de-DE.json [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+  "settings-option-enable-system-borders-details": "Erfordert einen Neustart",
+  "settings-option-enable-voice": "Aktiviere Sprachsteuerung",
+  "settings-option-enable-voice-details": "Sprachsteuerung für GPMDP, dies ist experimentell",
+
+  "settings-option-hotkey-explanation": "Die grundlegenden Mediatasten werden dennoch funktionieren.  Diese Einstellungen kommen zu den Standardeinstellungen hinzu.",
```
Comment (ChrisB9) [inline src/_locales/de-DE.json]: Mediatasten vielleicht zu Medientasten um Denglisch weitestgehend zu vermeiden?

**Comment by MarshallOfSound** [comment]
@ChrisB9 @bernikr @MetalMatze @berryelectronics @MCManuelLP 

Is this translation PR all set to go?  If you think it is simply comment below with ":+1:"

**Comment by MetalMatze** [comment]
I'll quickly update this PR later. There are some valid comments. Then I'm :+1:

**Comment by MetalMatze** [comment]
Ok, I'm good. 👍 
Rebased to the current `master` branch.

**Comment by MarshallOfSound** [comment]
This isn't rebased :confused: 

You have 47 commits, there should be one :confounded:

**Comment by Bonez76** [comment]
Is this completely reviewed? I could also have a look at the German version but I have no idea how to open the "de.pak" file I found in (.../AppData\Local\GPMDP_3\app-3.2.4\locales).

Could anyone assist me with this?

**Comment by MarshallOfSound** [comment]
@Bonez76 This PR has moved into #1084 and yes it is for the german translation of the UI

That `de.pak` file is for the Chromium translations not for the app UI translations

**Comment by Bonez76** [comment]
@MarshallOfSound Thanks for the quick reply. Where do I find the file to review/translate on a Windows 10 system? No clue :(

**Comment by MarshallOfSound** [comment]
@Bonez76 

https://github.com/MetalMatze/Google-Play-Music-Desktop-Player-UNOFFICIAL-/blob/f2c24abf18760e601d68b0ec6541596921db1595/src/_locales/de.json

**Comment by Bonez76** [comment]
@MarshallOfSound Thanks again! I am new to Github. Would love to help with this. Subscribed to #1084  already :)
